simcon_project/conversation_templates/views/conversation.py

```diff
@@ -199,9 +199,13 @@ def conversation_end(request, ct_response_id):
 
     # POST request
     if request.method == 'POST':
+        print(request.POST)
         for response in ct_node_responses:
             response.transcription = request.POST.get(str(response.id), '')
             response.save()
+        # The key has to be 0, I have no clue why, just don't touch it
+        ct_response.self_rating = request.POST.get('0', 0)
```

Comment:
aren't you getting the primary key 0?

Reply:
Only because anything else wouldn't work for some reason. Had help on discord and we couldn't figure it out otherwise